### ISSUE TITLE
Multiple 404 and redirect issues for taxonomy/category pages

### ISSUE THREAD

**Issue by bradleycamacho** [newrelic/docs-website#1475]
There are multiple areas in the taxonomy with strangely behaving redirects and 404 errors.  @roadlittledawn has fixed a good chunk of these, but there are still several dozen broken redirects and taxonomy links. 

These are in a private g-sheet. If you need access, ask me for "Hacky 404 sheet test"

This issue is pretty bare-bones right now, but I'll fill it in as I explore this today.

**Comment by austin-schaefer** [comment]
Relates to #1460

**Comment by mmfred** [comment]
accidentally closed (reopened now). Also, assigned to @bradleycamacho for now.

**Comment by jpvajda** [comment]
Is this different than what we are resolving in #1460 ?

**Comment by mmfred** [comment]
@jpvajda I think so. Now that that's merged, I've seen some of this is definitely fixed. I'll have the team check some known areas and we'll close this morning if this issue is no longer relevant.

**Comment by bradleycamacho** [comment]
Yes, this is slightly different. #1460 is about a dozen or so broken link in the left nav, which @roadlittledawn has now fixed. This is about ~100 redirects that are still throwing 404s.

I am taking care of this today. Updating the description now.

**Comment by jpvajda** [comment]
@bradleycamacho @mmfred  Thanks for the info! Is this something the Docs team is going to resolve manually?

**Comment by bradleycamacho** [comment]
@jpvajda Yes, I'm adding them to the site nav right now.

**Comment by austin-schaefer** [comment]
@bradleycamacho : LMK if this is big enough we should bring this into a sprint rather than a GH issue.

**Comment by bradleycamacho** [comment]
We're working on this as part of our current sprint. I'll keep this open for now and close once the work is complete.
